doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -275,7 +275,7 @@ From lowest to highest:
 The NGSIv2 specification describes in section "Subscriptions" the rules that trigger notifications
 corresponding to a given subscription, based on updates to the entities covered by the subscription.
 Apart from that kind of regular notifications, Orion may send also an initial notification at
-subscription creation/update time. Check details in the document about [initial notifications](initial_notification.md)
```

Comment:
Maybe is learer to leave this paragraph as it was and add the new _"Initial Notification can be configurable using URI parameter `skipInitialNotification`"_ in a separate paragraph. In addition and example can be include, e.g:

```
Initial Notification can be configurable using a new URI parameter option  `skipInitialNotification`. For instance `POST /v2/subscriptions?options=skipInitialNotification`.
```

(Note also the change "URL parameter" -> "URI parameter option" which is more aligned with other parts of documetnation)

Reply:
fixed in 89317a9